vmm_core/vmotherboard/src/chipset/backing/arc_mutex/device.rs

```diff
@@ -206,7 +206,6 @@ where
     /// Includes some basic validation that returns an error if a device
     /// attempts to use a service without also implementing the service's
     /// corresponding `ChipsetDevice::supports_` method.
-    #[allow(clippy::should_implement_trait)] // obviously not std::ops::Add
```

Comment:
So, by removing these, if we ever decide to enable these lints again then we'll have to reevaluate all of these removals.

Reply:
Yeah... but then again - how often do we actually look at the big book of clippy lints and decide that we'll enable one that's already existed for however many years?

More often than not, we only tweak our lint configurations alongside rust version bumps, and when we do so - we typically only consider the set of lints that have just been introduces / who's behavior has been changed.